fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -271,6 +302,12 @@ private void handleQuery() {
                 parsedStmt = stmts.get(i);
                 parsedStmt.setOrigStmt(new OriginStatement(originStmt, i));
 
+                // Only add the last running stmt for multi statement,
+                // because the audit log will only show the last stmt.
```

Comment:
How to handle the failure of `show frontends`?

Reply:
The failure of stmt will also be recorded at the end of handleQuery, because it has catched all exceptions.